code/demos/Simulation.ipynb

```diff
@@ -459,61 +499,14 @@
     "Based on how existing lightcurve fitters work, a lightcurve is generally assigned a class before its redshift and distance modulus are estimated because the fitting function will differ based on the assigned class.  Thus, we may assume that the lightcurve likelihood is separable as follows:\n",
     "\n",
     "\\begin{align*}\n",
-    "p(\\underline{\\ell}_{n} | t_{n}, z_{n}, \\mu_{n}, \\vec{P}, \\vec{V}, \\vec{C}) &\\propto \\frac{p(\\underline{\\ell}_{n} | t_{n})\\ p(\\underline{\\ell}_{n}, t_{n} | z_{n}, \\mu_{n})}{\\int\\ d\\underline{\\ell}_{n}\\ p(\\underline{\\ell}_{n} | \\vec{P}, \\vec{V}, \\vec{C})\\ p(\\underline{\\ell}_{n} | t_{n})\\ p(\\underline{\\ell}_{n}, t_{n} | z_{n}, \\mu_{n})}\n",
+    "\\ln[p(\\underline{\\ell}_{n} | t_{n}, z_{n}, \\mu_{n})] &= \\ln[p(\\underline{\\ell}_{n} | t_{n})] + \\ln[p(\\underline{\\ell}_{n}, t_{n} | z_{n}, \\mu_{n})]\n",
     "\\end{align*}"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "*The supernova lightcurve selection function*\n",
-    "\n",
-    "As a placeholder until we identify a better physical model, we assume a selection function represented by $\\ln[p(\\underline{\\ell}_{n} | \\vec{P}, \\vec{V}, \\vec{C})]$ with three contributions over the spaces of magnitudes in the filters populating the space $F$ and times relative to the peak of the lightcurve populating the space $S$.\n",
-    "\n",
```

Comment:
Remind me why we are placing an additional selection function on the LCs when we are assuming that the LCs generate the mus but here aren't using lcs directly?

Also, the main selection functions are given over probability, redshift and number, right?

Reply:
I'm confused because this snippet is from old code that does not appear in [branch 15](https://github.com/aimalz/scippr/blob/issue-15-selection/code/demos/Simulation.ipynb).